sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/sources/TextSocketMicroBatchStream.scala

```diff
@@ -220,43 +192,3 @@ class TextSocketMicroBatchReadSupport(options: DataSourceOptions)
 }
 
 case class TextSocketInputPartition(slice: ListBuffer[(UTF8String, Long)]) extends InputPartition
-
-class TextSocketSourceProvider extends DataSourceV2
```

Comment:
This is moved to a new file: https://github.com/apache/spark/pull/23430/files#diff-c223080834806efd92efc860656b60cdR34